README.md

```diff
@@ -274,6 +274,42 @@ tart clone acme.io/remoteorg/name:latest my-local-vm-name
 
 This invocation calls the `tart pull` implicitly (if the image is not being present) before doing the actual cloning.
 
+### Mounting directories
+
+To mount a directory, run the VM with the `--dir` argument:
+
+```
+tart run --dir=project:~/src/project vm
+```
+
+Here, the `project` specifies a mount name, whereas the `~/src/project` is a path to the host's directory to expose to the VM.
+
+It is also possible to mount directories in read-only mode by adding a third parameter, `ro`:
+
+```
+tart run --dir=project:~/src/project:ro vm
+```
+
+Note: to use the directory mounting feature, the host needs to run macOS 13.0 (Ventura) or newer.
+
+#### Accessing mounted directories in macOS guests
+
+All shared directories are automatically mounted to `/Volumes/My Shared Files` directory.
+
+The directory we've mounted above will be accessible from the `/Volumes/My Shared Files/project` path inside a guest VM.
+
+Note: to use the directory mounting feature, the guest VM needs to run macOS 13.0 (Ventura) or newer.
+
+#### Accessing mounted directories in Linux guests
+
+To be able to access the shared directories from the Linux guest, you need to manually mount the virtual filesystem first:
+
+```
+mount -t virtiofs com.apple.virtio-fs.automount /mnt/shared
+```
+
+The directory we've mounted above will be accessible from the `/mnt/shared/project` path inside a guest VM.
```

Comment:
does this mean that it's not possible to mount it to an arbitrary location, and that it must have the same name and be unique?

E.g. if I were to try to mount 2 folders named `public_html` in different locations that are not `/mnt/shared` would that be possible?

Reply:
> E.g. if I were to try to mount 2 folders named `public_html` in different locations that are not `/mnt/shared` would that be possible?

I don't think so, you can only mount the "outer" directory that contains all of the "inner" mounts specified by `--dir`.

Reply:
No, you can use `--dir` multiple times, this is explicitly stated in`tart run --help`.

Reply:
can that be included in the readme with an example? How does it handle when multiple dir are added but they have the same folder name e.g. `~/project1/www`  and `/project2/www`?

Reply:
> can that be included in the readme with an example?

See https://github.com/cirruslabs/tart/pull/338/commits/7a26b3b4781288d9a7dd859ec97bce01b56a957e.